content/doc/tutorials/tutorial-for-installing-jenkins-on-IBM-Cloud.adoc

```diff
@@ -17,18 +17,18 @@ At the end of the tutorial you will have a cluster with Jenkins up and running.
 
 ## Step 1 provision Kubernetes Cluster
 
-* Click the **Catalog** button on the top
+* Select the **Catalog** button on the top
```

Comment:
Hi @kellie-freeman thanks very much for alerting me on this, since the larger section headings are numbered already, and these are presented as the "main" steps at the top of the page, would it be better to use a different list styling for these?